teachertool/src/components/TabGroup.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from "react";
+// eslint-disable-next-line import/no-internal-modules
+import css from "./styling/TabGroup.module.scss";
+
+import { useContext } from "react";
+import { classList } from "react-common/components/util";
+import { Button } from "react-common/components/controls/Button";
+import { AppStateContext } from "../state/appStateContext";
+import { TabName } from "../types";
+import { setActiveTab } from "../transforms/setActiveTab";
+
+interface ITabProps extends React.PropsWithChildren<{}> {
+    name: TabName;
+}
+
+export const Tab: React.FC<ITabProps> = ({ children, name }) => {
+    const { state: teacherTool } = useContext(AppStateContext);
+
+    const onClick = () => {
+        setActiveTab(name);
+    };
+
+    const active = teacherTool.activeTab === name;
+    const classes = classList("tt-tabgroup-tab", active ? "tt-tabgroup-tab-active" : undefined);
+
+    return (
+        <Button className={classes} title={name} onClick={onClick}>
+            {children}
```

Comment:
Why would a button/tab have children? I understand the use case for the TabGroup and the TabPanel, but I'm a little confused about the use here.

Reply:
This is how react-common is designed. It can receive button content like label as a child element. We may want to utilize this flexibility to show a progress indicator on the results tab while the eval is running, or show an indicator that the results have updated.